tutorials/xr/xr_action_map.rst

```diff
@@ -385,3 +385,124 @@ This is why many XR runtimes only use it as a last resort and will attempt to us
   This is our advice as well: limit your action map to the interaction profiles for devices you have actually tested your game with.
   The Oculus Touch controller is widely used as a fallback controller by many runtimes.
   If you are able to test your game using a Meta Rift or Quest and add this profile there is a high probability your game will work with other headsets.
+
+Binding Modifiers
+-----------------
+
+While one of the main goals of the action map is to remove the need for the application to know the hardware used, sometimes the hardware has physical differences that require inputs to be altered in ways other than how they are bound to actions. This need ranges from setting thresholds, to altering the inputs available on a controller.
```

Comment:
> While one of the main goals of the action map is to remove the need for the application to know the hardware used, sometimes the hardware has physical differences that require inputs to be altered in ways other than how they are bound to actions.

This is a complex sentence that took multiple attempts to parse for me as a non-native speaker. I suggest breaking it up a bit or simplifying if possible :)

Reply:
```suggestion
One of the main goals of the action map is to remove the need for the application to know the hardware used. However, sometimes the hardware has physical differences that require inputs to be altered in ways other than how they are bound to actions. This need ranges from setting thresholds, to altering the inputs available on a controller.
```
Quickest fix is to use "however". There's probably an even better fix though

Reply:
As a non-native speaker, I'm struggling to put this in the right words for sure :)

The main thing is that just binding inputs to actions isn't enough to cater for the physical differences between devices, further modifications to the inputs are required per device. Hence the binding modifier system being introduced.